time send_time
uint32 msg_type

uint32 value_1
float32 value_2
